packages/trees/frontend/sirius-components-trees/src/views/getTreeEventSubscription.ts

```diff
@@ -10,11 +10,11 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-export const getTreeEventSubscription = (depth: number): string => {
+export const getTreeEventSubscription = (depth: number, eventType: string, eventInputType: string): string => {
```

Comment:
From what I've seen in the code, if I'm right, you will have to duplicate this subscription to have a getExplorerEventSubscription in `sirius-web-application` instead and just keep the function `recursiveGetChildren` and the fragment here...

Reply:
Done.